<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyProfile</class>
 <widget class="QDialog" name="MyProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedPlus - My Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #e3f2fd, stop:1 #ffffff);
    border-radius: 15px;
}</string>
  </property>
  <widget class="QFrame" name="frameCircle1">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>-80</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(25, 118, 210, 0.15), stop:1 transparent);
    border-radius: 150px;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameCircle2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>500</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(66, 165, 245, 0.15), stop:1 transparent);
    border-radius: 175px;
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 32px;
    font-weight: bold;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:0.5 #42a5f5, stop:1 #64b5f6);
    padding: 20px 30px;
    border-radius: 15px;
    border: 2px solid rgba(255, 255, 255, 0.3);
    letter-spacing: 1px;
    qproperty-alignment: AlignCenter;
}</string>
   </property>
   <property name="text">
    <string>üë§ My Profile</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameProfileCard">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>800</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: white;
    border-radius: 15px;
    border: 2px solid #e0e0e0;
}</string>
   </property>
   <widget class="QFrame" name="frameAvatar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #42a5f5, stop:1 #1e88e5);
    border-radius: 80px;
    border: 5px solid white;
}</string>
    </property>
    <widget class="QLabel" name="lblAvatar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 80px;
    color: white;
    background: transparent;
    border: none;
    qproperty-alignment: AlignCenter;
}</string>
     </property>
     <property name="text">
      <string>üë§</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>700</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 28px;
    font-weight: bold;
    color: #1976d2;
    background: transparent;
    qproperty-alignment: AlignCenter;
}</string>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPatientId">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>700</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 14px;
    color: #757575;
    background: transparent;
    qproperty-alignment: AlignCenter;
    font-style: italic;
}</string>
    </property>
    <property name="text">
     <string>Patient ID: ---</string>
    </property>
   </widget>
   <widget class="QFrame" name="lineDiv">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>700</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: #e0e0e0;
    border: none;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmailIcon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 20px;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>üìß</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmail">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: #37474f;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>Email: ---</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPhoneIcon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 20px;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>üì±</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPhone">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: #37474f;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>Phone: ---</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAddressIcon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 20px;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>üè†</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAddress">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: #37474f;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>Address: ---</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditProfile">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 16px;
    font-weight: bold;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #66bb6a, stop:1 #43a047);
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #4caf50, stop:1 #388e3c);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #388e3c, stop:1 #2e7d32);
}</string>
    </property>
    <property name="text">
     <string>‚úèÔ∏è Edit Profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>680</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 16px;
    font-weight: bold;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:1 #42a5f5);
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1565c0, stop:1 #1976d2);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #0d47a1, stop:1 #1565c0);
}</string>
   </property>
   <property name="text">
    <string>‚Üê Back to Dashboard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>